### PR TITLE
[BULK UPDATE] DocuTune - Rebranding

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort for the Microsoft Learn platform rebrand**

The Microsoft team is applying changes to content for the Microsoft Learn platform rebrand. Changes include modifications to current Microsoft Learn and Microsoft Docs terminology as well as associated domain name and content URL path changes.

Please review this draft PR and provide any necessary comments as soon as possible. Because these fixes are tightly scoped, we have approval to automatically merge these changes to the default branch for publication.

This PR was generated via DocuTune. It includes only targeted fixes and minor formatting adjustments.

CorrelationId: d8c4c3ac-d49a-4c95-9ddc-9822ca297907
InitiativeId: docutune-rebranding-2022-09
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Office Add-ins documentation
 
-Welcome to the Office Add-ins documentation repo. In this repository, you can find the documentation source files for Office JavaScript API concepts, quick starts, tutorials, and how-to guides. For the best experience, we recommend you view this content on the [Microsoft Docs](https://learn.microsoft.com/office/dev/add-ins) site.
+Welcome to the Office Add-ins documentation repo. In this repository, you can find the documentation source files for Office JavaScript API concepts, quickstarts, tutorials, and how-to guides. For the best experience, we recommend you view the [Office Add-ins documentation](https://learn.microsoft.com/office/dev/add-ins) on Microsoft Learn.
```
Comment (ElizabethSamuel-MSFT) [inline README.md:3]:
```suggestion
Welcome to the Office Add-ins documentation repo. In this repository, you can find the documentation source files for Office JavaScript API concepts, quick starts, tutorials, and how-to guides. For the best experience, we recommend you view the [Office Add-ins documentation](https://learn.microsoft.com/office/dev/add-ins) on Microsoft Learn.
```

Comment (alexbuckgit) [reply README.md:3]: Style guide recommendation is one word for "quickstart".
https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=28014

Comment (ElizabethSamuel-MSFT) [reply README.md:3]: That's not the style we're currently using in our Office docs.

Comment (ElizabethSamuel-MSFT) [reply README.md:3]: Office Style Guide uses "quick start": https://styleguides.azurewebsites.net/StyleGuide/Read?id=2665

**Comment by annmariehitchcock** [comment]
#docutune-ready

**Pull request merged**